lib/temporal/worker/activity_worker.rb

```diff
@@ -0,0 +1,103 @@
+require 'temporal/error/failure'
+require 'temporal/worker/sync_worker'
+require 'temporal/worker/activity_runner'
+
+module Temporal
+  class Worker
+    class ActivityWorker
+      def initialize(task_queue, core_worker, activities, converter, executor)
+        @running = false
+        @task_queue = task_queue
+        @worker = SyncWorker.new(core_worker)
+        @activities = prepare_activities(activities)
+        @converter = converter
+        @executor = executor
+        @running_activities = {}
+      end
+
+      def run(reactor)
+        @running = true
+
+        while running?
```

Comment:
This will change when you implement shutdown.

Reply:
Yep, it is expected 👍